3.2/root/usr/bin/run-mongod-pet

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Run mongod as part of PetSet.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${CONTAINER_SCRIPTS_PATH}/common.sh
+
+function usage() {
+  echo "You must specify the following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "  MONGODB_KEYFILE_VALUE"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "Optional variables:"
+  echo "  MONGODB_SERVICE_NAME (default: mongodb)"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
```

Comment:
> @rhcarvalho @php-coder OpenShift will support to somehow use data created by old version in petset deployment?

IMHO we shouldn't. Petset is a new thing and I assume that someone who will try it, use it from scratch. Also, if user wants to use old data he/she can provide custom config file.

@bparees Please, approve/disprove it.